index.html

```diff
@@ -25,7 +25,22 @@
         }
 
         .container {
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            min-height: 100vh;
             max-width: 600px;
+            padding: 20px;
+        }
+
+        .cards {
+            flex-grow: 1;
+            display: flex;
+            flex-direction: column;
+            justify-content: flex-start;
+            align-items: center;
+            width: 100%;
+            overflow-y: auto;
```

Comment:
To ensure that the scoring section remains at the bottom of the view, even when new cards are added, you can modify the CSS and HTML structure. First, adjust the CSS for the .container and add a new .cards class: